lib/unifile-fs.js

```diff
@@ -32,6 +33,19 @@ class FsConnector {
     this.name = NAME;
   }
 
+  resolvePath(path) {
+    // is the path alreay resolved
+    if(path.startsWith(Os.homedir()) && Path.resolve(path).startsWith(Os.homedir()))
```

Comment:
The FsConnector already have a `rootPath` property to resolve against. It defaults to the first sandbox path if given or `/`.